### DISCUSSION TITLE
ly2: find parse node from compile node?

### DISCUSSION THREAD

**Discussion by choppsv1** [CESNET/libyang#1407] [Q&A]
Is there a way to lookup a `lysp_node_container` that corresponds to an `lysc_node_container`?

**Comment by michalvasko** [comment]
No, this is not possible. It may be added in future if it is needed but we are hoping to avoid that because you would essentially need the whole compilation code to do this correctly.

**Reply by choppsv1** [reply]
I was thinking if the compiled node had a back-reference to the parse node I could check lysp->presence to see if the container was actually a presence container or if it was changed to this by libyang internals.

**Reply by michalvasko** [reply]
I saw your suggestion in the issue and realized it makes sense but I have not implement it yet so I did not reply. Please be patient, I am absolutely swamped with issues and am supposed to even do some development. I will always reply to every issue, eventually.

**Reply by choppsv1** [reply]
#1408 :)

**Reply by choppsv1** [reply]
looks like i need to attend to the unit tests for the PR sitll.

**Reply by choppsv1** [reply]
ok all fixed.
